rapid7_insightidr/komand_rapid7_insightidr/util/parse_dates.py

```diff
@@ -0,0 +1,40 @@
+from dateutil.parser import parse, ParserError
+from komand_rapid7_insightidr.util.relative_time_codes import relative_time_to_milliseconds
+from komand.exceptions import PluginException
+import time
+import logging
+
+
+def parse_dates(time_from_string: str, time_to_string: str, relative_time_from: str) -> (int, int):
```

Comment:
```suggestion
def parse_dates(time_from_string: str, time_to_string: Optional[str], relative_time_from: str) -> (int, int):
```
Also add `from typing import Optional` to the imports above for this